defs/bugs/juju/juju_core.yaml

```diff
@@ -17,4 +17,14 @@ input:
       post-series-upgrade failed, issues with unit {} - see LP bug for
       workaround.
     search-result-format-groups: [1]
-
+1852502:
+  input:
```

Comment:
There is a another way to do this which will allow the check to also work on a live host as well as sosreport:

input:
type: command
value: journalctl
meta:
kwargs:
unit: juju-db

Reply:
made this change, thank you.